metrics-kamon/src/main/scala/com/ovoenergy/effect/Kamon.scala

```diff
@@ -16,16 +16,16 @@ object Kamon {
 
     def counter(metric: Metric): F[Long => F[Unit]] =
       Sync[F]
-        .delay(K.counter(metric.name).refine(metric.tags))
+        .delay(K.counter(metric.name).withTags(TagSet.from(metric.tags)))
         .map(
-          counter => times => Sync[F].delay(counter.increment(times))
+          counter => times => Sync[F].delay{ counter.increment(times); () }
```

Comment:
how come this change?

Reply:
Because in the new version of Kamon, `counter.increment` returns the counter, instead of returning unit. That triggers a warning about a non-unit value being discarded (and we treat warnings as errors). This change makes it explicit that we're discarding the value returned from `increment`, so the compiler is happy.

Reply:
I know I am joining too late, but wouldn't something like this be more readable / clear about the intention?

```
Sync[F].delay(counter.increment(times)).map(_ => ())
```

Reply:
or even `Sync[F].delay(counter.increment(times)).as(()))`

Reply:
I wonder if the 2.13 break is the time to finally remove these Kamon modules :eyes:

Reply:
Thanks :) See #27